libkernelflinger/trusty_efi.c

```diff
@@ -377,7 +378,8 @@ static EFI_STATUS start_tos_image(IN VOID *bootimage)
                         (UINTN)load_base + tos_header->entry_offset);
         debug(L"Call TOS loader entry_addr = 0x%x", call_entry);
         tos_ret = call_entry(startup_info_v2);
-
+        OPENSSL_cleanse(startup_info_v2->seed_list, sizeof(startup_info_v2->seed_list));
```

Comment:
Why not put this under "cleanup:". Wouldn't that be logical place

Reply:
If error happens in line 355 seed_list or rpmb_key will not be "scrubbed"

Reply:
done